scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -1429,6 +1431,65 @@ while true; do
     fi
   fi
 
+      # Mithril Signer Section
+      SERVICE_EXISTS=$(systemctl list-units --type=service --all | grep -q 'cnode-mithril-signer.service'; echo $?)
```

Comment:
Could be moved outside of loop to not have to check this on every refresh. Its also hardcoded for `cnode` name, please use `${CNODE_VNAME}` instead. 

Not a must but can check how cncli service is checked in same script to not have to pipe and grep. Though I guess fine like this as well. 
```
if command -v cncli >/dev/null && command -v systemctl >/dev/null && systemctl is-active --quiet ${CNODE_VNAME}-cncli-sync.service 2>/dev/null; then
```

Reply:
Hello,

I used ${CNODE_VNAME}, about "systemctl is-active" this only return if is active but not the sub-statuses of active: running, waiting, exited